lib/portage/util/__init__.py

```diff
@@ -2006,3 +2008,12 @@ def read_ld_so_conf(path):
     rval.append("/lib")
 
     return [normalize_path(x) for x in rval if x]
+
+
+def no_color(settings: Optional[dict]) -> bool:
+    """This will enable us to safely clean up support for NOCOLOR"""
```

Comment:
This is still vague/missing the point, it also shouldn't be a docstring.

```suggestion
    # In several years (2026+), we can cleanup NOCOLOR support, and just support NO_COLOR.
```